WordPress/Classes/Services/AccountService+MergeDuplicates.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+extension AccountService {
+
+    func mergeDuplicatesIfNecessary() {
+        let accounts = allAccounts()
+        guard accounts.count > 1 else {
```

Comment:
One last nitpick, for performance. Since most of the times this will not happen, and `allAccounts` still has to fetch all the accounts, I'd suggest to use `numberOfAccounts() > 1` for this check. This uses `countForFetchRequest` instead of `executeFetchRequest`, which should be much faster, as it doesn't have to read all the data from disk.

Reply:
Ooooo good call.